### PR TITLE
Update docker with internal changes

### PR DESCRIPTION
**Pull request by nzlz**

### CODE REVIEW

**Review by LanderU — APPROVED** [review]
LGTM!

File: docker/README.md [hunk 1]
```diff
@@ -15,11 +15,12 @@ docker build -t r2l .
 
 ```shell
 cd ~/ros2learn/docker
-docker rm r2l || true && docker run -it --name=r2l -h ros2learn -v `pwd`/tmp:/tmp r2l
+docker rm r2l || true && docker run -it --name=r2l -h ros2learn -v `pwd`/tmp:/tmp/ros2learn r2l
```
Comment (LanderU) [inline docker/README.md:18]: We should change the instructions in gym-gazebo2 as well.

File: docker/README.md [hunk 2]
```diff
@@ -15,11 +15,12 @@ docker build -t r2l .
...
 
 #Inside the docker container, used to load visual models
 cp -r /root/ros2_mara_ws /tmp
```
Comment (nzlz) [inline docker/README.md]: Shouldn't this line be `cp -r /root/ros2_mara_ws tmp/ros2learn` ?

Comment (LanderU) [reply docker/README.md]: Yes!

Comment (nzlz) [reply docker/README.md]: Indeed we needed this. Commited the change.

**Pull request merged**